packages/cw-derive/examples/basic.rs

```diff
@@ -0,0 +1,30 @@
+#![allow(dead_code)]
+use cosmwasm_std::Response;
+
+struct Ctx;
```

Comment:
In a real contract, Ctx would have a defined structure, and Error would be some custom `this_error` type, right?

Reply:
`Ctx` would be a tuple of `(DepsMut, Env, MessageInfo)` (or `Deps` for queries) or something which can be made from it with `.into()`. Here it is only for generation purposes.

Error would be any error type returned by contract, in particular it can be one generated with `this_error` (in our codebase it will be, but it is not enforced by codegen).